### ISSUE TITLE
[Request] Random delay from moving

### ISSUE THREAD

**Issue by SmilingSword** [plgonzalezrx8/MagicPoGoBot#3]
#random stop from moving
Random stop from moving but still catching pokemon if sighted. 
Stop like few second delay 1-3 on every min between (10-30)  (take a break anytime)

**Comment by plgonzalezrx8** [comment]
Good Idea, working on this as well. If any developer wants to help me with this one, I'll approve the push.
